web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v280/MoveHarvesterSettingsToHigherNumber.java

```diff
@@ -47,12 +44,18 @@ private void loadChildren(Statement statement) throws SQLException {
             }
         }
 
-        public void write(Statement statement) throws SQLException {
-            final String sql = format("INSERT INTO " + getHarvesterSettingsName() + " (id, parentId, name, value) VALUES (%s, %s, " +
-                                      "'%s', '%s')", id, parentId, name, value);
-            statement.execute(sql);
+        public void write(Connection connection) throws SQLException {
+            final String sql = format("INSERT INTO %s (id, parentId, name, value) VALUES (?, ?, " +
+                                      "?, ?)", getHarvesterSettingsName());
+            try(PreparedStatement statement = connection.prepareStatement(sql)) {
```

Comment:
It is more for the use of parameters and parameter handling instead of concatenating strings and using string.format